packages/neo4j-driver/test/examples.test.js

```diff
@@ -828,8 +828,8 @@ describe('#integration examples', () => {
 
     const people = await result.toPromise()
     expect(people).toEqual([
-      Notification.createNext('Infinity Gauntlet'),
       Notification.createNext('Mjölnir'),
+      Notification.createNext('Infinity Gauntlet'),
```

Comment:
This change won't solve the issue and also make it more easy to happen. We need a equal comparison which not depends on the order, actually. But for now, we can revert this change.